### PR TITLE
Update to NEMS Roles

### PR DESCRIPTION
**Pull request by andrewbolster**
Updating Ops Manual to realistically reflect current and upcoming NEMS roles: 

Of particular note is the dissolution of Projects NEM, being folded into the Project Room NEM, as well as the dissolution of Facilities position and the creation of an Outreach Nem

### CODE REVIEW

**Comment by chris18890** [comment]
Looks grand to me :+1:

**Comment by mhazley** [comment]
I like these changes. Well done. 

I also want to add something though - I think that both the Project Room & Projects roles should be expanded to have a point to drive the layout and function of the room. My thinking here is that we need to drive this as the room is in 'limbo' and needs direction. 

Suggested point (for both roles):

 - Drives the function and layout of the Project Room to suit the needs of the Farset Community. 
 
In light of that, I then think that the Project Room role should be expanded to drive room changes that have been agreed on by NEMS and Directors.

Suggested Point (for Project Room role):

 - Organises and drives Project Room changes.

File: about/ops_manual.md [hunk 1]
```diff
@@ -72,24 +72,33 @@ Removal of NEMs from their positions is a Directorship-level decision. A vote of
...
+* Projects:
+  * PoC for project storage, resources, and documentation
+  * Attempting to prevent conflict between member projects
+  * Driving project documentation and PR.
+  * Along with the Projects Room NEM, rives the function and layout of the Project Room to suit the needs of the Farset Community.
+* Project-Room:
+  * Management and monitoring of Project equipment and resources including but not limited to 3D printer.
+  * Attempting to prevent conflict between member project resource requirements.  
+  * Coordinates Project Room changes
+  * Along with the Projects NEM, rives the function and layout of the Project Room to suit the needs of the Farset Community.
```
Comment (chris18890) [inline about/ops_manual.md]: typo - *drives

File: about/ops_manual.md [hunk 2]
```diff
@@ -72,24 +72,33 @@ Removal of NEMs from their positions is a Directorship-level decision. A vote of
...
-  * Partially responsible for ensuring storage is kept reasonably organised.
+* Community: 
+  * PoC for internal community engagement
+  * Responsible for internal social events and member relations
+  * PoC for member induction and 'soft landing' discussions
+* Projects:
+  * PoC for project storage, resources, and documentation
+  * Attempting to prevent conflict between member projects
+  * Driving project documentation and PR.
+  * Along with the Projects Room NEM, rives the function and layout of the Project Room to suit the needs of the Farset Community.
```
Comment (chris18890) [inline about/ops_manual.md]: typo - *drives

**Comment by mhazley** [comment]
@clairebones can you please merge this because we have all agreed that this is the way forward. 

@andrewbolster at the next directors meeting can you action a change to add new roles to the site? This will be:

 - Community - Claire
 - Project Room - Colin
 - Projects - Matt

Thanks!

**Pull request merged**